File: PRIMERAPREG/python/primero.py
# -*- coding: utf-8 -*-
"""
Created on Fri Dec  3 21:58:11 2021

@author: USUARIO
"""
def calculadora(a,b,op):
    if (op == "suma"):
        return suma(a,b)
    elif (op == "resta"):
        return resta(a,b)
    elif (op == "multiplicacion"):
        return mutilplicacion(a,b)
    else:
        return division(a,b)
def suma (a,b):
    return a+b    
    
def resta (a,b):
    return a-b    
def multiplicacion (a,b):
    return a*b    
def division (a,b):
    return a/b        
    
    
def main():
    print ("calculadora")
    print ("Ingrese primer numero")
    a = input ()
    print ("Ingrese otro numero")
    b = input ()
    print ("Ingrese la operacion que desea realizar")
    op = input ()
    print(calculadora(int(a),int(b),op))
main()



    
    

File: PRIMERAPREG/Haskell/primero.hs
module Test where
-- This tells main to do nothing.
suma :: Int -> Int -> Int
suma a b = a + b;

resta :: Int -> Int -> Int
resta a b = a - b;

multip :: Int -> Int -> Int
multip a b = a * b;

divi :: Int -> Int -> Int
divi a b = a * b;

calculadora :: Int -> Int -> String -> Int
calculadora a b "suma" =  suma a b
calculadora a b "multi" =  multip a b  
calculadora a b "resta" =  resta a b
calculadora a b "divi" =  divi a b 

 
